src/goto-symex/goto_symex.h

```diff
@@ -80,6 +80,39 @@ class goto_symext
     const goto_functionst &goto_functions,
     const goto_programt &goto_program);
 
+  /// Symexes from the first instruction and the given state, terminating as
+  /// soon as the last instruction is reached.  This useful to explicitly symex
+  /// certain ranges of a program, e.g. in an incremental decision procedure.
+  /// \param state Symex state to start with.
+  /// \param goto_functions GOTO model to symex.
+  /// \param first Entry point in form of a first instruction.
+  /// \param last Final instruction, which itself will not be symexed anymore.
```

Comment:
If this is the first one to *not* symex then rename "limit", describe "Instruction after the last to symex, or instructions.end() if symex should continue to end of function"